### YamlMime:ManagedReference
items:
- uid: System.Workflow.Activities.Rules.RuleEvaluationException
  id: RuleEvaluationException
  children:
  - System.Workflow.Activities.Rules.RuleEvaluationException.#ctor
  - System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String)
  - System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RuleEvaluationException
  nameWithType: RuleEvaluationException
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException
  type: Class
  summary: "表示所有因規則評估問題所造成的例外狀況的基底類別。"
  syntax:
    content: 'public class RuleEvaluationException : System.Workflow.Activities.Rules.RuleException'
  inheritance:
  - System.Exception
  - System.Workflow.Activities.Rules.RuleException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor
  id: '#ctor'
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  langs:
  - csharp
  name: RuleEvaluationException()
  nameWithType: RuleEvaluationException.RuleEvaluationException()
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException()
  type: Constructor
  assemblies:
  - System.Workflow.Activities
  namespace: System.Workflow.Activities.Rules
  summary: "初始化的新執行個體<xref href=&quot;System.Workflow.Activities.Rules.RuleEvaluationException&quot;></xref>類別。"
  remarks: "這個建構函式初始化<xref:System.Exception.Message%2A>屬性的新執行個體的系統提供的訊息會描述錯誤並考量目前系統文化特性。</xref:System.Exception.Message%2A>       下表顯示<xref:System.Workflow.Activities.Rules.RuleEvaluationException>.</xref:System.Workflow.Activities.Rules.RuleEvaluationException>的執行個體的初始屬性值      |屬性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|Null 參考 (`Nothing`在 Visual Basic 中)。 |  |<xref:System.Exception.Message%2A>|系統提供的當地語系化描述。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuleEvaluationException ();
    parameters: []
  overload: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  langs:
  - csharp
  name: RuleEvaluationException(String)
  nameWithType: RuleEvaluationException.RuleEvaluationException(String)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(String)
  type: Constructor
  assemblies:
  - System.Workflow.Activities
  namespace: System.Workflow.Activities.Rules
  summary: "初始化的新執行個體<xref href=&quot;System.Workflow.Activities.Rules.RuleEvaluationException&quot;></xref>類別指定的錯誤訊息。"
  remarks: "這個建構函式初始化<xref:System.Exception.Message%2A>屬性的新執行個體使用`message`參數。</xref:System.Exception.Message%2A>       下表顯示<xref:System.Workflow.Activities.Rules.RuleEvaluationException>.</xref:System.Workflow.Activities.Rules.RuleEvaluationException>的執行個體的初始屬性值      |屬性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|Null 參考 (`Nothing`在 Visual Basic 中)。 |  |<xref:System.Exception.Message%2A>|錯誤訊息字串。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuleEvaluationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "描述錯誤的訊息。"
  overload: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  langs:
  - csharp
  name: RuleEvaluationException(SerializationInfo,StreamingContext)
  nameWithType: RuleEvaluationException.RuleEvaluationException(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Workflow.Activities
  namespace: System.Workflow.Activities.Rules
  summary: "初始化的新執行個體<xref href=&quot;System.Workflow.Activities.Rules.RuleEvaluationException&quot;></xref>使用序列化資料的類別。"
  remarks: "重新建構所需的資料流上傳輸的例外狀況物件的還原序列化期間呼叫這個建構函式。"
  syntax:
    content: protected RuleEvaluationException (System.Runtime.Serialization.SerializationInfo serializeInfo, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: serializeInfo
      type: System.Runtime.Serialization.SerializationInfo
      description: "<xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref>其中保存有關擲回例外狀況的序列化的物件資料。"
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "<xref:System.Runtime.Serialization.StreamingContext>其中包含關於來源或目的端內容資訊。</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  langs:
  - csharp
  name: RuleEvaluationException(String,Exception)
  nameWithType: RuleEvaluationException.RuleEvaluationException(String,Exception)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(String,Exception)
  type: Constructor
  assemblies:
  - System.Workflow.Activities
  namespace: System.Workflow.Activities.Rules
  summary: "初始化的新執行個體<xref href=&quot;System.Workflow.Activities.Rules.RuleEvaluationException&quot;></xref>類別，使用指定的錯誤訊息以及<xref:System.Exception>造成此<xref:System.Exception>.</xref:System.Exception>的</xref:System.Exception>內部參考"
  remarks: "<xref:System.Exception>前一次的直接結果擲回<xref:System.Exception>應該包含在先前的例外狀況的參考<xref:System.Exception.InnerException%2A>屬性。</xref:System.Exception.InnerException%2A> </xref:System.Exception> </xref:System.Exception> <xref:System.Exception.InnerException%2A>屬性會傳回相同的值傳遞到建構函式，則為 null 參考 (`Nothing`) 如果<xref:System.Exception.InnerException%2A>屬性沒有提供內部<xref:System.Exception>建構函式的值。</xref:System.Exception> </xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       下表顯示<xref:System.Workflow.Activities.Rules.RuleEvaluationException>.</xref:System.Workflow.Activities.Rules.RuleEvaluationException>的執行個體的初始屬性值      |屬性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|內部<xref:System.Exception>參考。 |  |<xref:System.Exception.Message%2A>|錯誤訊息字串。 |</xref:System.Exception.Message%2A> </xref:System.Exception> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuleEvaluationException (string message, Exception ex);
    parameters:
    - id: message
      type: System.String
      description: "描述錯誤的訊息。"
    - id: ex
      type: System.Exception
      description: "<xref:System.Exception>也就是目前<xref:System.Exception>.</xref:System.Exception>的原因</xref:System.Exception> 如果`innerException`參數不是 null 參考 (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref>在 Visual Basic 中)，<xref:System.Exception>可在處理內部<xref:System.Exception>.</xref:System.Exception> catch 區塊中引發</xref:System.Exception>目前"
  overload: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Workflow.Activities.Rules.RuleException
  isExternal: false
  name: System.Workflow.Activities.Rules.RuleException
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException()
  nameWithType: RuleEvaluationException.RuleEvaluationException()
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException()
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String)
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException(String)
  nameWithType: RuleEvaluationException.RuleEvaluationException(String)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException(SerializationInfo,StreamingContext)
  nameWithType: RuleEvaluationException.RuleEvaluationException(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String,System.Exception)
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException(String,Exception)
  nameWithType: RuleEvaluationException.RuleEvaluationException(String,Exception)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException
  nameWithType: RuleEvaluationException.RuleEvaluationException
